README.md

```diff
@@ -72,4 +39,4 @@ Give a ⭐️ if you like this project!
 
 This project is [MIT](./LICENSE) licensed.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [OPTIONAL] I noticed that you added the MIT.md template to your project 💯👌, but you forgot to customize it, kindly add your name and customize it. 👌
- [OPTIONAL] Also, the MIT.md link in your README is broken, kindly fix it by changing the path to ./MIT.md instead of  ./LINCENSE 👍 and you can remove this _NOTE (the last two lines) from your README I think It's not necessary 👌